util/binfmt_misc/detect.go

```diff
@@ -0,0 +1,32 @@
+package binfmt_misc
+
+import (
+	"sync"
+
+	"github.com/containerd/containerd/platforms"
+)
+
+var once sync.Once
+var arr []string
+
+func SupportedPlatforms() []string {
+	once.Do(func() {
+		def := platforms.DefaultString()
+		arr = append(arr, def)
+
+		if p := "linux/amd64"; def != p && amd64Supported() {
+			arr = append(arr, p)
+		}
+		if p := "linux/arm64"; def != p && arm64Supported() {
+			arr = append(arr, p)
+		}
+		if p := "linux/arm/v7"; def != p && armSupported() {
```

Comment:
what happens if def == `linux/arm/v6` ?

Reply:
Then we don't add anything but that to platforms